app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<br/>
+<h1><%= t("manger_user.list_user") %></h1>
+<div class="col-md-10 col-lg-offset-1">
+  <table class="table table-bordered">
+    <tr class="bg-primary">
+    <th class="text-center col-md-1"><%= I18n.t("manger_user.index") %></th>
+    <th class="text-center col-md-2"><%= t("manger_user.full_name") %></th>
+    <th class="text-center col-md-3"><%= t("manger_user.email") %></th>
+    <th class="text-center col-md-4"><%= t("manger_user.activated") %></th>
+    <th class="text-center col-md-5"><%= t("manger_user.activated_at") %></th>
+    <th class="text-center col-md-6"></th>
+    <th class="text-center col-md-6"></th>
+  </tr>
+    <% @users.each_with_index do |item, index| %>
+        <tr>
+          <td><%= create_index params[:page], index, Settings.user.page %></td>
+          <td scope="row"><%= item.full_name %></td>
+          <td scope="row"><%= item.email %></td>
+          <td scope="row"><%= item.activated %></td>
+          <td scope="row"><%= item.activated_at %></td>
+          <td scope="row"><%= link_to t("manger_user.delete"), item, method: :delete %></td>
+          <td scope="row"><%= link_to t("manger_user.show"), item %></td>
+        </tr>
+    <% end %>
```

Comment:
Tìm hiểu RENDER PARTIALS IN RAILS 
https://viblo.asia/p/render-partials-in-rails-ZjleadVEGqJ